Backend/admin/src/components/login/login.js

```diff
@@ -3,60 +3,74 @@ import { Button, Form, Input } from 'antd';
 import { useHistory } from 'react-router';
 import { login } from '../../db/auth';
 
+import './login.css';
+
 const layout = {
-  labelCol: { span: 8 },
+  labelCol: { span: 6 },
   wrapperCol: { span: 16 },
 };
 const tailLayout = {
-  wrapperCol: { offset: 8, span: 16 },
+  wrapperCol: { offset: 6, span: 16 },
 };
 
 const Login = () => {
   const history = useHistory();
 
+  const [form] = Form.useForm();
+
   const onFinish = ({ email, password }) => {
     login(email, password)
       .then(() => {
         console.log('Success:');
         history.push('/questions');
       })
-      .catch(() => console.log('display login fail error message'));
+      .catch(() => {
+        form.resetFields();
+        console.log('display login fail error message');
+      });
   };
 
   const onFinishFailed = (errorInfo) => {
     console.log('Failed:', errorInfo);
   };
 
   return (
-    <Form
-      {...layout}
-      name="basic"
-      initialValues={{ remember: true }}
-      onFinish={onFinish}
-      onFinishFailed={onFinishFailed}
-    >
-      <Form.Item
-        label="Email"
-        name="email"
-        rules={[{ required: true, message: 'Please input your email!' }]}
+    <div className="login-wrapper">
+      <Form
+        {...layout}
+        name="basic"
+        initialValues={{ remember: true }}
+        onFinish={onFinish}
+        onFinishFailed={onFinishFailed}
+        className="login-form"
+        style={{ padding: '24px' }}
+        form={form}
       >
-        <Input />
-      </Form.Item>
+        <Form.Item
+          label="Email"
+          name="email"
+          rules={[{ required: true, message: 'Please input your email!' }]}
+        >
+          <Input />
+        </Form.Item>
 
-      <Form.Item
-        label="Password"
-        name="password"
-        rules={[{ required: true, message: 'Please input your password!' }]}
-      >
-        <Input.Password />
-      </Form.Item>
-
-      <Form.Item {...tailLayout}>
-        <Button type="primary" htmlType="submit">
-          Submit
-        </Button>
-      </Form.Item>
-    </Form>
+        <Form.Item
+          label="Password"
+          name="password"
+          rules={[{ required: true, message: 'Please input your password!' }]}
+        >
+          <Input.Password />
+        </Form.Item>
+        <Form.Item {...tailLayout} st>
+          <Button type="primary" htmlType="submit">
+            Submit
+          </Button>
+          <a className="login-form-forgot" href="/">
```

Comment:
@RafaelClaudiu  @razvanip  I thought it will be nice to have forgot pass button, what do you think?

Reply:
Will be nice. Sure. In this PR or open a new one just for the password. If you want

Reply:
I will open another PR, if it's ok i can merge this

Reply:
Go for it